### ISSUE TITLE
Low contast on book page navigation

### ISSUE THREAD

**Issue by mfairchild365** [unlcms/unl_four#22]
Book pages are failing WCAG2.0AA

example: http://ucommfairchild.unl.edu/unlcms/test-book
